cypress/integration/instance/InstanceCreation.ts

```diff
@@ -0,0 +1,126 @@
+import { onlyOn } from "@cypress/skip-test";
+import {
+  createInstance,
+  deleteInstance,
+  safeLogin,
+  uniqueName,
+  isEnvironmentType,
+  EnvType,
+  pageWasLoaded,
+} from "../../utils/Util";
+
+const newInstanceName: string = uniqueName("new-instance");
+
+describe("the 'Create a SE instance' Modal", () => {
+  beforeEach(() => {
+    cy.visit("/");
+    safeLogin();
+    pageWasLoaded();
+  });
+
+  it("Submit", () => {
+    createInstance(newInstanceName);
+
+    //Assert that the process of the instnace's creation was started
+    cy.ouiaId("Instances list table", "PF4/Table")
+      .ouiaId(newInstanceName, "PF4/TableRow")
+      .should("be.visible")
+      .within(() => {
+        //The user cannot go to the instance's details if it is not ready
+        //Assert that the inner text contains only the name of the instance and no <a>
+        cy.get("td:first").should("have.html", newInstanceName);
+        //The status column is loading and contains ready
+        cy.get("td:nth-child(3)")
+          .ouiaId("accepted", "QE/ResourceStatus")
+          .within(() => {
+            cy.get("span[role='progressbar']").should("exist");
+            cy.ouiaId("creating", "PF4/Button").should("have.text", "Creating");
+            cy.ouiaId("ready-shortly", "QE/HelperTextItem").should(
+              "have.text",
+              "This will be ready shortly."
+            );
+          });
+        cy.ouiaId("creating", "PF4/Button").click();
+      });
+
+    //Assert that the process of the instnace's creation is monitored
+    cy.ouiaId("se-status", "QE/Popover")
+      .should("be.visible")
+      .within(() => {
+        cy.ouiaType("QE/StackItem").should("have.length", "3");
+        cy.ouiaId("info-banner", "QE/StackItem")
+          .ouiaId("ready-shortly", "PF4/Text")
+          .should("be.visible");
+        cy.ouiaId("steps-count", "QE/StackItem").should(
+          "have.text",
+          "0 of 3 steps completed"
+        );
+        //TODO: There should be a check that all steps are met but I did not know how to do it for Mocked Env
+        //It seems that '1 of 3' is skipped and `2 of 3` is here only for few secs.
+        //Each step is asserted with the mocked version
```

Comment:
I think we can improve the mocked API handler to also include the skipped `preparing` status. Could you please create a JIRA and link it in this comment?

Reply:
Yes ... https://issues.redhat.com/browse/MGDOBR-1118